src/OrchardCore.Modules/OrchardCore.Layers/Services/DefaultLayersMethodProvider.cs

```diff
@@ -69,8 +71,19 @@ public DefaultLayersMethodProvider()
                         : string.Equals(requestPath, url, StringComparison.OrdinalIgnoreCase); ;
                 })
             };
+
+            _currentCulture = new GlobalMethod
+            {
+                Name = "currentCulture",
+                Method = serviceProvider => (Func<string, object>)(culture =>
+                {
+                    var currentCulture = CultureInfo.CurrentCulture.Name.ToLowerInvariant();
+
+                    return string.Equals(culture.ToLowerInvariant(), currentCulture, StringComparison.OrdinalIgnoreCase);
```

Comment:
Should it check for the segments? like `fr-CA` would be a match with `fr`. To do this use the `Parent` property too.